thefuck/rules/rbenv_install.py

```diff
@@ -0,0 +1,14 @@
+from thefuck.shells import shell
+from thefuck.utils import for_app
+
+
+@for_app('rbenv')
+def match(command):
+    return ('ruby-build: definition not found' in command.output.lower() and
+            'if the version you need is missing, try upgrading ruby-build' in command.output.lower()
```

Comment:
Could the `.lower()` parts be removed?

From the few times I tried, I saw that the output is consistent as follows:
```
ruby-build: definition not found: 2.6.6

See all available versions with `rbenv install --list'.

If the version you need is missing, try upgrading ruby-build:

  git -C /root/.rbenv/plugins/ruby-build pull
```

Apart from the upgrade command. BTW, `rbenv` should have an `upgrade` command.